pynest/nest/lib/hl_api_exceptions.py

```diff
@@ -77,34 +77,32 @@ class NESTError(Exception):
         """Base exception class for all NEST exceptions.
         """
 
-        def __init__(self, message, *args, **kwargs):
+        def __init__(self, message):
             """Initializer for NESTError base class.
 
             Parameters:
             -----------
             message: full error message to report.
```

Comment:
```suggestion
            message: str
                full error message to report.
```

Reply:
Just to double-check: this should always be a str, not a tuple, correct?

Reply:
That's right.